eventbrite.py

```diff
@@ -0,0 +1,430 @@
+            "name": {
+                "html": f"<p>{title}</p>"
+            },
+            "description": {
+                "html": f"<p>{description}</p>"
+            },
+            "start": {
+                "timezone": "UTC",
+                "utc": date_start.strftime("%Y-%m-%dT%H:%M:%SZ")
+            },
+            "end": {
+                "timezone": "UTC",
+                "utc": date_end.strftime("%Y-%m-%dT%H:%M:%SZ")
+            },
+            "currency": "AUD",
+            "online_event": False,
+            "organizer_id": "",
+            "listed": True,
+            "shareable": True,
+            "invite_only": False,
+            "show_remaining": True,
+            "capacity": 100,
+            "is_reserved_seating": False,
+            "is_series": False,
+            "show_pick_a_seat": False,
+            "show_seatmap_thumbnail": False,
+            "show_colors_in_seatmap_thumbnail": False,
+            "locale": "en_AU"
+        }
+    }
+
+    # Convert the event details to a JSON string
+    event_detail_json = json.dumps(event_detail)
+
+    # Create the event
+    new_event = create_event(http,eventbrite_api_token, organization_id, event_detail_json)
+
+    # Extract the event_id from the response
+    event_id = new_event.get('id')
+
+    # Create the ticket class details JSON
+    ticket_detail = {
+        "ticket_class": 
+        {
+            "name": "API Test1",
+            "donation": "false",
+            "quantity_total": "100",
+            "minimum_quantity": "1",
+            "maximum_quantity": "10",
+            "hidden": "false",
+            "auto_hide": "false",
+            "auto_hide_before": "",
+            "auto_hide_after": "",
+            "sales_channels": ["online","atd"],
+            "hide_sale_dates": "false",
+            "delivery_methods": ["electronic"],
+            "include_fee": "false",
+            "sales_start": date_start.strftime("%Y-%m-%dT%H:%M:%SZ"),
+            "sales_end": date_end.strftime("%Y-%m-%dT%H:%M:%SZ"),
+            **({"cost": f"AUD,{cost_cents}", "free": "false"} if cost_cents is not None else {'free': "true"}),
+        }
+    }
+
+    # Convert the ticket class details to a JSON string
+    ticket_detail_json = json.dumps(ticket_detail)
+
+    # Create the ticket class
+    new_tickets = create_ticket_class(http, eventbrite_api_token, event_id, ticket_detail_json)
+    if publish:
+        publish_event(http, eventbrite_api_token, event_id)
+    return new_event, new_tickets
+
+if __name__ == "__main__":
+
+    # Retrieve the values of an environment variables
+    eventbrite_api_token = os.environ['EVENTBRITE_API_TOKEN']
+    organization_id = os.environ['EVENTBRITE_ORGANIZATION_ID']
+    publish = os.environ.get("PUBLISH")
+
+    # Create a PoolManager object
+    http = urllib3.PoolManager()
+
+    quick_create_event(http, eventbrite_api_token, organization_id, "API-TEST-04", "API-TEST-DESCRIPTION-4","2023-11-12T14:00:00Z", 2, 2000, publish)
```

Comment:
I'm skeptical that the date works if I pass 5pm perth time. eg. "2023-11-12T17:00:00+0800"

Reply:
I want further additional 'timezone' argument to the `quick_create_event`  function, and I want date_start to be timezone UNAWARE. However the quick_create_event code will combine the time unaware `date_start` and `timezone` argument to be timezone aware datetime object, and then you can convert that to UTC if you want, but it looks like the eventbrite API has a timezone field for dates anyway.

Example timezone in IANA format:  'Australia/Perth'